### DISCUSSION TITLE
Add an option to copy screenshots to the clipboard without saving them as files

### DISCUSSION THREAD

**Discussion by SergioRibera** [YaLTeR/niri#901] [Ideas]
Currently, taking a screenshot in Niri always saves the image as a file, which can lead to unnecessary accumulation of files. This makes it tedious to manually select which screenshots to keep and which to delete.  

I propose adding an option in the screenshot UI to copy the image directly to the clipboard without saving it as a file. This would enhance the workflow, especially for users who only need temporary access to screenshots, and reduce the need for manual cleanup.  

**Expected benefits**:  
- Minimized file accumulation.  
- Improved usability of the screenshot system.  
- Faster workflows for tasks that only require temporary image sharing.

attached evidence 😞 
![image](https://github.com/user-attachments/assets/0ad2a850-b277-497b-99e9-16980da452cb)

**Comment by YaLTeR** [comment]
Maybe a special bind like Ctrl+C to just copy and not save to disk?

If you don't want to save any screenshots to disk btw you can set `screenshot-path null`

**Reply by extsy0** [reply]
> Maybe a special bind like Ctrl+C to just copy and not save to disk?
> 
> If you don't want to save any screenshots to disk btw you can set `screenshot-path null`

That bind would be good! Saving to disk by default is nice but sometimes you don't need it

**Reply by SergioRibera** [reply]
Exactly, the idea is not to remove saving to file, but to have both options, sometimes I just want to copy and other times I want to copy and save

**Comment by SergioRibera** [comment]
Thanks, this discussion is closed by https://github.com/YaLTeR/niri/pull/975
